introduction/installation/python-bindings.md

```diff
@@ -6,4 +6,269 @@ parent: Installation
+query doesn't matter (e.g. when creating a table or inserting data into a
+table).
+
+```python
+import glaredb
+con = glaredb.connect()
+
+# Create a table.
+con.execute("create table my_table (a int)")
+# Insert some data.
+con.execute("insert into my_table values (1), (2)")
+
+# Query the table we just created. Note that we're using `sql` here because we
+# want to show the results.
+con.sql("select * from my_table").show()
+
+# ┌───────┐
+# │ a     │
+# │ ──    │
+# │ Int32 │
+# ╞═══════╡
+# │ 1     │
+# │ 2     │
+# └───────┘
+```
+
+## Example
+
+{: .important}
+
+> For more examples, see <https://github.com/GlareDB/glaredb/tree/main/py-glaredb/examples>
+
+The following is an example that joins a polars data frame with data from a 
+hosted demo Postgres instance.
+
+```python
+import glaredb
+import polars as pl
+
+# Polars DataFrame created in the script
+df = pl.DataFrame(
+    {
+        "region": [0, 1, 2, 3, 4],
+        "population": [10, 20, 30, 40, 50]
+    }
+)
+
+con = glaredb.connect()
+
+# Join the above Polars DataFrame on data from our demo Postgres instance
+result = con.sql(
+        """select
+            t1.r_regionkey,
+            t1.r_name,
+            t2.Population
+        from
+            read_postgres('postgres://demo:demo@pg.demo.glaredb.com/postgres', 'public', 'region') as t1
+        join
+            df as t2 on t1.r_regionkey = t2.region;"""
+).to_polars();
+
+print(result)
+```
+
+The above results in:
+
+```console
+shape: (5, 3)
+┌─────────────┬───────────────────────────┬────────────┐
+│ r_regionkey ┆ r_name                    ┆ population │
+│ ---         ┆ ---                       ┆ ---        │
+│ i32         ┆ str                       ┆ i64        │
+╞═════════════╪═══════════════════════════╪════════════╡
+│ 3           ┆ EUROPE                    ┆ 40         │
+│ 0           ┆ AFRICA                    ┆ 10         │
+│ 2           ┆ ASIA                      ┆ 30         │
+│ 4           ┆ MIDDLE EAST               ┆ 50         │
+│ 1           ┆ AMERICA                   ┆ 20         │
+└─────────────┴───────────────────────────┴────────────┘
+```
+
+## API
+
+- `module glaredb`
+  - `connect([data_dir], [spill_path]): LocalSession`
+    - `data_dir` optional - if provided, data will be stored at this path,
+      otherwise all data is in memory.
+    - `spill_path` optional - a path where glaredb can create temporary files.
+      If not provided, a random temporary directory is used.
+- `class LocalSession`
+  - `prql(query): PyLogicalPlan`
+    - Lazily evaluates the `query` string (PRQL syntax)
+  - `sql(query): PyLogicalPlan`
+    - Lazily evaluates the `query` string (SQL syntax)
+  - `execute(query): PyExecutionResult`
+    - Eagerly evaluates the `query` string (SQL syntax)
+  - `close()`
+- `class PyLogicalPlan`
+  - `to_arrow()`
+  - `to_pandas()`
+  - `to_polars()`
+  - `show()`
+- `class PyExecutionResult`
+  - `to_arrow()`
+  - `to_pandas()`
+  - `to_polars()`
+  - `show()`
```

Comment:
(**optional**): Would you be down to drop this from the installation - it's incomplete/outdated and I'm not sure it's useful here. I defer to you!